### ISSUE TITLE
getPropertyType and SVGScriptElement href baseVal property

### ISSUE THREAD

**Issue by lukewarlow** [w3c/trusted-types#521]
Currently getPropertyType's spec and both Chromium and WebKit's implementation has no handling of the .href.baseVal property of an SVGScriptElement, but it does require a TrustedScriptURL.

cc @koto what should we do here?

**Comment by koto** [comment]
Realistically we only have two choices, either ignore it (as `baseVal` is a property of `SVGAnimatedString` and not an element), or use a meta syntax like `href.baseVal` for the property name and add an entry for it in [getPropertyType](https://w3c.github.io/trusted-types/dist/spec/#dom-trustedtypepolicyfactory-getpropertytype). 

I have a slight preference to the former - getPropertyType does not cover all TT sinks, and we should just assume that SVG animations are another vector that needs to be handled separately.

**Comment by lukewarlow** [comment]
Yeah I have a preference for ignoring it too. Same way lots of functions require a trusted type without a metadata API.

**Comment by fred-wang** [comment]
cc @smaug----
